v5_client_web_socket_service.go

```diff
@@ -25,6 +25,7 @@ func (s *V5WebsocketService) Public(category CategoryV5) (V5WebsocketPublicServi
 	return &V5WebsocketPublicService{
 		client:              s.client,
 		connection:          c,
+		category:            category,
```

Comment:
Since category is taken as an argument when initializing service, it is not strange if service holds category.